wiki/Forums/en.md

```diff
@@ -0,0 +1,76 @@
+# Forums
+
+![osu!forums](main.jpg "Image of the osu!forums")
+
+The osu!forums is one of the main features of the osu!web.
```

Comment:
🤔 I'm not sure... there is only one fourm but when talking about the fourms as a whole, then you would add the `s` to fourm?